File: src/Modelo/Cliente.java

package Modelo;


public class Cliente {
    private String nombreapellido;
    private String direccion;
    private int contacto;

    public Cliente(String nombreapellido, String direccion, int contacto) {
        this.nombreapellido = nombreapellido;
        this.direccion = direccion;
        this.contacto = contacto;
    }

    public int getContacto() {
        return contacto;
    }

    public void setContacto(int contacto) {
        this.contacto = contacto;
    }

    public String getNombreapellido() {
        return nombreapellido;
    }

    public void setNombreapellido(String nombreapellido) {
        this.nombreapellido = nombreapellido;
    }

    public String getDireccion() {
        return direccion;
    }

    public void setDireccion(String direccion) {
        this.direccion = direccion;
    }

    @Override
    public String toString() {
        return "Cliente{" + "nombreapellido=" + nombreapellido + ", direccion=" + direccion + ", contacto=" + contacto + '}';
    }
    
    
}

File: Modelo/Cliente.cs
using System;

namespace Woodic.Modelo
{
    public class Cliente
    {
        public long Contacto { get; set; }
        public string Nombre { get; set; } = string.Empty;
        public string Direccion { get; set; } = string.Empty;

        public Cliente() { }

        public Cliente(long contacto, string nombre, string direccion)
        {
            Contacto = contacto;
            Nombre = nombre;
            Direccion = direccion;
        }

        // Métodos de compatibilidad
        public long getContacto() => Contacto;
        public void setContacto(long contacto) => Contacto = contacto;
        public void setContacto(int contacto) => Contacto = contacto;

        public string getNombre() => Nombre;
        public void setNombre(string nombre) => Nombre = nombre;

        public string getDireccion() => Direccion;
        public void setDireccion(string direccion) => Direccion = direccion;
    }
}
